src/main/kotlin/ch/srgssr/pillarbox/monitoring/event/EventDispatcherClientConfiguration.kt

```diff
@@ -0,0 +1,26 @@
+package ch.srgssr.pillarbox.monitoring.event
+
+import ch.srgssr.pillarbox.monitoring.event.config.RetryProperties
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.NestedConfigurationProperty
+
+/**
+ * Configuration class for Server-Sent Events (SSE) client settings in the application.
+ * This class is mapped to properties prefixed with `pillarbox.monitoring.dispatch` in
+ * the application's configuration files.
+ *
+ * @property uri The URI for the SSE endpoint. Defaults to "http://localhost:8080".
```

Comment:
There doesn't seem to be a default value for `uri`.
```suggestion
 * @property uri The URI for the SSE endpoint.
```